starfish/filters.py

```diff
@@ -142,17 +115,3 @@ def bin_thresh(img, thresh):
     """
     res = img >= thresh
     return res
-
-
-def white_top_hat(img, disk_size):
-    """
-    Performs white top hat filtering of an image to enhance spots
-    :param img: Image to filter. :type numpy array
-    :param disk_size: Radius of the disk-shaped structuring element.  :type int
-    :return: Filtered image, same shape as input :type ndarray
-    """
-    selem = disk(disk_size)
-    min_filt = minimum_filter(img, footprint=selem)
-    max_filt = maximum_filter(min_filt, footprint=selem)
-    res = img - max_filt
```

Comment:
doesn't this need a res = WhiteTopHatFilter.filter(img, disk_size) ?

Reply:
oh sorry -- looks like you're actually nuking this in favor for the new python api, but are leaving gaussian_high_pass in the old python api?